CorsixTH/Luatest/spec/entities/humanoid_spec.lua

```diff
@@ -23,10 +23,16 @@ require("class_test_base")
 require("entity")
 require("entities.humanoid")
 
+assert:register("assertion", "matches", assertion_matches, "assertion.matches")
```

Comment:
This I am still not happy about, but I am not sure how to solve it. It would be best to register the assertion inside the class_test_base, but the assert table isn't defined there (I don't know why:() Any Ideas?

Reply:
Awesome it worked! Thanks man. How come i don't need to require luassert inside the test classes? Is it some internal magic?